searchtool/templates/searchtool/choose_a_location.html

```diff
@@ -0,0 +1,52 @@
+{% extends "search.html" %}
+{% load static %}
+{% load wagtailcore_tags %}
+{% block body_class %}template-choosealocation{% endblock %}
+
+{% block extra_css %}
+
+{% endblock extra_css %}
+
+{% block content %}
+<div class="searchtool-section choose-a-location">
+  <div class="searchtool-header">
+    {{ page.header|richtext }}
+  </div>
+  <div class="tab-container">
+    <ul class="tab-headings">
+      {% for block in page.tab_headings %}
+        {% if forloop.first %}
+          <li tabindex="0" role="button" class="selected">{{block}}</li>
+          {% else %}
+          <li tabindex="0" role="button">{{block}}</li>
+        {% endif %}
+      {% endfor %}
+    </ul>
+    <ul id="Country" class="country-options">
+      {% for block in page.country_options %}
+        {% if forloop.last %}
+          <li tabindex="0" role="button" id="NorthernIreland">{{block}}</li>
```

Comment:
It might make more sense for the id to be called something other then NorthernIreland as it could change in the future, maybe something generic like `LastCountryOption`?